docs/_includes/contributors.html

```diff
@@ -55,10 +55,14 @@ <h1 class="section-heading text-uppercase">{{ site.data.sitetext.contributors.ti
     <div class="client-slide fiserv1">
 			<img src="/assets/img/contributors/fiserv.svg" height="50" width="120" alt="Fiserv" />
 		</div>
+	<div class="svg-space"></div>
```

Comment:
Could this code be formatted so it's easier to read?

Reply:
@soham4abc Any update on this?

Reply:
Do you want me to remove the extra spaces?

Reply:
Do you want me to remove the spaces?